config/routing.go

```diff
@@ -24,8 +24,9 @@ var (
 	// Default filter-protocols to pass along with delegated routing requests (as defined in IPIP-484)
 	// and also filter out locally
 	DefaultHTTPRoutersFilterProtocols = getEnvOrDefault("IPFS_HTTP_ROUTERS_FILTER_PROTOCOLS", []string{
-		"unknown", // allow results without protocol list, we can do libp2p identify to test them
+		//"unknown", // allow results without protocol list, we can do libp2p identify to test them
```

Comment:
nit: we need to include `unknown` – Amino DHT does not store information about supported protocols, and if we don't include `unknown` we will not get any results if something other than `cid.contact` is used.

```suggestion
		"unknown", // allow results without protocol list, we can do libp2p identify to test them
```

Context: if you look at https://delegated-ipfs.dev/routing/v1/providers/bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi `transport-bitswap` is returned only by IPNI, where it is hardcoded at announcement time. Amino DHT results don't have `Protocols` field, thus we want to always include `unknown` to ensure `delegated-ipfs.dev`, self-hosted https://github.com/ipfs/someguy (or any other [`/routing/v1`](https://specs.ipfs.tech/routing/http-routing-v1/) implementation)  can be used instead of `cid.contact` backed by IPNI.

Reply:
Ops sorry, this wasn't intended